### DISCUSSION TITLE
out redirection

### DISCUSSION THREAD

**Discussion by stefanardo** [cesanta/mongoose#2856] [Ask questions, get answers] [answered]
I am developing a web server with STM32H, I have a custom board, how can I redirect out to uart8?

Thanks and regards.

Stefano

**Comment by scaprile** [comment, marked as answer]
Please don't open an issue to ask a question.
Converted to question.
Supported USARTs are those that output data to a useful port in those supported boards; usually the port connected to the USB to serial interface in that board. If, for some reason, you want to use a different USART, then:
- read your device user manual
- open hal.h in the existing example for your device
- add setup code for your desired UART in your device
- #define UART_DEBUG yourdesiredUART

**Reply by stefanardo** [reply]
I am sorry, I've fixed. 
Thanks and regards.
